index.html

```diff
@@ -242,6 +244,52 @@ <h4>Skills</h4>
                 </div>
             </div>
         </section>
+        <!--Contact form section-->
+        <section class="contact-me">
+            <!--contact me section main text-->
+            <div class="contactme-text">
+                <h1 class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h1>
```

Comment:
- [ ] Kindly use **h3 or h4 or h5** instead of **h1** because it is not the main heading of your portfolio website. The `h1` tag is specifically designed for main headings, providing a clear and prominent representation of the page's primary topic or purpose. By utilizing the *h3 or h4 or h5* tag, you create a visual hierarchy that helps users quickly identify and determine the relevance of the content to their needs and interests. This improves the overall usability and navigation of your website, ensuring a more engaging and effective experience for visitors.